components/mixins/experience-event/industry-verticals/experienceevent-travel-preferences.schema.json

```diff
@@ -0,0 +1,208 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/travel-preferences",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Travel Preferences",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["travel_and_hospitality"]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
```

Comment:
Should be extending the EE class right ?

Reply:
No ,it should be profile.

Reply:
In that case this has to be moved to profle folder under mixins. Its currently under experience-event/industry-verticals